addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -584,6 +584,14 @@ export const editorCommands = {
         }
 
         let target = [...(blocks.size ? blocks : li)];
+        if (blocks.size) {
+            // Remove the hardcoded padding from email templates
```

Comment:
This is not the right comment because it is specific to a particular use case, which is neither true nor adequate. It is not adequate because it's weird to write code in `web_editor` if it is really only for another module. Sometimes you don't have the choice, and this might be one of those times, but the thing is it's not even actually true. It might be true now, but you might very well have the same issue anywhere else in Odoo where there would be similar paddings at some point. Basically, the issue is not about email templates but about padding. However, the information that those padding were seen in email template is useful information as well. That's where commit messages can shine and be ultra specific to the practical issue we encountered.

Reply:
Makes sense, I updated the comment